### PR TITLE
feat: move the website project into the website package

### PR DESCRIPTION
**Pull request by NilsJacobsen**
see: https://github.com/opral/inlang.com/issues/74

### CODE REVIEW

**Comment by NilsJacobsen** [comment]
@NiklasBuchfink can you remove paraglide from the editor package? There is a dependency on the website now that we don't want to have. It also causes problems on build time.

I would say as long as we do not dogfood the editor we should get completely rid of it in the package. Don't want to mess up the editor. Can you take over?

**Comment by NiklasBuchfink** [comment]
@NilsJacobsen Sadly, it is not done in a minute. I tried it before, but I can take over.

**Comment by NilsJacobsen** [comment]
No worries there is no extreme urge.

**Comment by NiklasBuchfink** [comment]
@NilsJacobsen Paraglide is removed from Fink ✅

File: inlang/source-code/editor/src/interface/components/Card.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { showToast } from "./Toast.jsx"
 import { rpc } from "@inlang/rpc"
 import { Button } from "#src/interface/components/Button.jsx"
 import Link from "#src/renderer/Link.jsx"
-import * as m from "#src/paraglide/messages.js"
```
Comment (NilsJacobsen) [inline inlang/source-code/editor/src/interface/components/Card.tsx:9]: Can we get rid of those? (Just use the hardcoded ones)

File: inlang/source-code/editor/src/interface/components/NewsletterForm.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { createSignal } from "solid-js"
 import { showToast } from "./Toast.jsx"
 import { rpc } from "@inlang/rpc"
-import * as m from "#src/paraglide/messages.js"
```
Comment (NilsJacobsen) [inline inlang/source-code/editor/src/interface/components/NewsletterForm.tsx:4]: Same here.

File: inlang/source-code/editor/src/interface/components/SearchBar.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import IconSearch from "~icons/material-symbols/search-rounded"
 import { currentPageContext } from "#src/renderer/state.js"
 import { createSignal } from "solid-js"
 import { navigate } from "vike/client/router"
-import * as m from "#src/paraglide/messages.js"
```
Comment (NilsJacobsen) [inline inlang/source-code/editor/src/interface/components/SearchBar.tsx:5]: Same here.

File: inlang/source-code/editor/src/renderer/Link.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { languageTag, sourceLanguageTag } from "#src/paraglide/runtime.js"
```
Comment (NilsJacobsen) [inline inlang/source-code/editor/src/renderer/Link.tsx:1]: We should get rid of the runtime import here too.

**Review by NilsJacobsen — COMMENTED** [review]
I think there are still some paraglide imports in here:)

**Comment by NilsJacobsen** [comment]
Thanks a lot @NiklasBuchfink let's merge this on Monday. I expect some missing translations on the website so we shouldn't push it out without time (weekend).

**Pull request merged**